[I18N] Fixed typos in pt-BR translation

TehPeGaSuS opened the pull request:
@RocketChat/android

Fixed typo related to PT/BR language regarding `gender` type of room

TehPeGaSuS committed:
Update strings.xml

Fixed typo related to PT/BR language regarding `gender` type of room

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/RocketChat/Rocket.Chat.Android?pullRequest=1186) <br/>All committers have signed the CLA.

rafaelks reviewed (approved):
Thank you!!! 👏

TehPeGaSuS committed:
Update strings.xml

I've overlooked this ones:
1.  `<string name="action_login_or_sign_up">` - In portuguese makes more sense
2. `<string name="msg_invalid_server_url">` - Wrong `gender` type
3. `<string name="permission_pinning_not_allowed">` edited to be in concordance with `<string name="message_pinned">`

TehPeGaSuS committed:
Update strings.xml

After talking to @rafaelks, we've decided that these two strings should be translated like this.
Using "Mutar" & "Desmutar", in some way, reminded me of mutations.

leonardoaramaki commented on app/src/main/res/values-pt-rBR/strings.xml:
```diff
@@ -59,7 +59,7 @@
     <string name="msg_invalid_2fa_code">Código 2FA inválido</string>
     <string name="msg_invalid_file">Arquivo inválido</string>
-    <string name="msg_invalid_server_url">URL de servidor inválida</string>
+    <string name="msg_invalid_server_url">URL de servidor inválido</string>
```
I think this was correct before. Wouldn't it be "**A** URL está inválida"?

TehPeGaSuS replied:
Well, bear with me as I'm portuguese and it makes more sense to me that way. :smile:

leonardoaramaki replied:
Oh, do you say "O URL" instead there?

TehPeGaSuS replied:
Yes, we do. That's why it makes more sense to me that way. But feel free to use your gramatic and semantic  there. :smile:

leonardoaramaki commented on app/src/main/res/values-pt-rBR/strings.xml:
```diff
@@ -122,7 +122,7 @@
     <string name="permission_editing_not_allowed">Edição não permitida</string>
     <string name="permission_deleting_not_allowed">Remoção não permitida</string>
-    <string name="permission_pinning_not_allowed">Fixar não permitido</string>
+    <string name="permission_pinning_not_allowed">Pinagem não permitida</string>
```
I think the word "Pinagem" doesn't exist in Portuguese in this sense, unless you're referring to this [https://www.dicio.com.br/pinagem/](url)

TehPeGaSuS replied:
Well, it's basically in that context yes. Like we used to use pins in the school to put leaves in cork boards.

leonardoaramaki replied:
Yeah, I think that's fine. :)

TehPeGaSuS replied:
:+1:

leonardoaramaki commented on app/src/main/res/values-pt-rBR/strings.xml:
```diff
@@ -162,8 +162,8 @@
-    <string name="Mute_someone_in_room">Mutar alguém</string>
-    <string name="Unmute_someone_in_room">Desmutar alguém na sala</string>
+    <string name="Mute_someone_in_room">Silenciar alguém</string>
+    <string name="Unmute_someone_in_room">De-silenciar alguém na sala</string>
```
I wish we had a proper antonym for "Silenciar" in PT_BR. Since we used "Mutar" and "Desmutar" before which I think does not exist, IMHO your fix is more accurate.

TehPeGaSuS replied:
The only option I'm currently seeing is to use `Remover silenciamento de alguém na sala`.
But i think this `new word` makes sense too :smiley:
